app/helpers/vm_infra_helper/textual_summary.rb

```diff
@@ -1,4 +1,5 @@
 module VmCloudHelper::TextualSummary
```

Comment:
@h-kataria Not for this PR, but shouldn't this be `VmInfraHelper`?  Is this file even needed if the constant has been wrong since 2013?